update the creating-tic page (#1876)
Samsung/tizen-docs: docs/platform/developing/creating-tic.md

2. Run the TIC docker container as a service:

   ```shell
      mkdir -p $HOME/tic-artifacts
      docker login -u [DockerHub_Username] -p [DockerHub_Password]
      docker-compose pull
      docker-compose up -d
   ```

### Connect to the TIC web-service

   * Open the web browser and enter `http://127.0.0.1:8088` in the address bar. The TIC web UI screen appears:

> 

Reviewer: ```suggestion
```

    ![TIC web-service](./media/tic-ui.png)

### Create a new Tizen image and SDK package

#### Settings
   * Easy
     * Select predefined set of images
     * Click `Apply changes` button
     ![Setting-Easy](./media/tic-ui.png)

   * Advanced
     * Select `Architecture`

>      * Import your own `Recipe`, `Repository & Package` (optional)

Reviewer: ```suggestion
     * Import your own `Recipe, Repository & Package` (optional)
```

     * Click `Apply changes` button

>      ![Setting-Advanced](./media/tic-settings-advanced.png)

Reviewer: This image is missing from the `docs/platform/developing/media` folder and hence doesn't show up in the staging site.

#### Packages
   * If you select `Easy` mode, in Settings stage, you can see `Summary` of the image to be generated.

>       ![Packages-Summary](./media/tic-packages-summary.png)

Reviewer: This image is missing from the `docs/platform/developing/media` folder and hence doesn't show up in the staging site.

   * If you select `Advanced` mode, in Settings stage, you can see `Details` tab.

>    ![Packages-Details](./media/tic-packages-details.png)

Reviewer: This image is missing from the `docs/platform/developing/media` folder and hence doesn't show up in the staging site.

>      * Select `packages` to be included in the image to be create.

Reviewer: ```suggestion
     * Select `packages` to be included in the image that will be created.
```

   * Click `Create` button

>    ![Packages-Create](./media/tic-packages-create.png)

Reviewer: This image is missing from the `docs/platform/developing/media` folder and hence doesn't show up in the staging site.

>      * Enter an image name

Reviewer: ```suggestion
     * Enter an image name.
```

>      * Enter an Image name

Reviewer: ```suggestion
     * Enter an image name
```

>      * Enter an Image name

Reviewer: ```suggestion
     * Enter an Image name.
```

     * Check `Generate SDK` checkbox, if you want to generate SDK package.
   * Click `Create` button again.

#### Jobs
   * You can see the list of image creation jobs and their progress.

>    ![Jobs-Progress](./media/tic-jobs-progress.png)

Reviewer: This image is missing from the `docs/platform/developing/media` folder and hence doesn't show up in the staging site.

   * Click `X` button if you want to cancel the job.

#### Images

>    * You can see the list of images

Reviewer: ```suggestion
   * You can see the list of images.
```

>    ![Images-List](./media/tic-images-list.png)

Reviewer: This image is missing from the `docs/platform/developing/media` folder and hence doesn't show up in the staging site.

>    * Click the link of the Image or SDK to download them.

Reviewer: ```suggestion
   * Click the link of the image or SDK to download them.
```

To flash the generated image by TIC to a target device for verification, see [Flashing an Image to RPI](flashing-rpi.md).

> To use the generated SDK to develope applications with Tizen Studio, see [Configuring the Package Manager](../../application/tizen-studio/setup/advanced-configuration.md) and [Configurable SDK](../../application/tizen-studio/configurable-sdk/configurable-sdk.md)

Reviewer: ```suggestion
To use the generated SDK to develope applications with Tizen Studio, see [Configuring the Package Manager](../../application/tizen-studio/setup/advanced-configuration.md) and [Configurable SDK](../../application/tizen-studio/configurable-sdk/configurable-sdk.md).
```
